tests/test_resolve.py

```diff
@@ -88,3 +91,14 @@ def test_if_does_not_fire_exception():
 
 def test_firstof_does_not_fire_exception():
     render(req('get'), template_name='test_firstof_does_not_fire_exception.html')
+
+def test_if_gitignore_has_migrations():
```

Comment:
think of a better name, bellow as well, and perhaps for the function itself, maybe just `validate_gitignore()`